casepro/backend/tests/test_junebug.py

```diff
@@ -613,6 +613,7 @@ def hub_outgoing_callback(request):
                     "user_id": "C-002",
                     "helpdesk_operator_id": self.user1.id,
                     'inbound_channel_id': '',
+                    "message_id": 8,
```

Comment:
I'm not sure that we are always guaranteed that these message IDs will be the same.. Maybe we should get them from the outgoing message object that we create?

Reply:
is this line the object we create? ```self.assertEqual(data, {"to": "+1234", "from": "+4321", "content": "That's great"})```

Reply:
It will be the `out_msg = self.create_outgoing(....` where we create the outbound. For this test it's on line 642